docs/content/mongocxx-v3/installation/windows.md

```diff
@@ -92,7 +61,7 @@ cd mongo-cxx-driver-r3.8.0/build
 Make sure you change to the `build` directory of whatever source tree you
 obtain.
 
-### Step 4: Configure the driver
+### Step 3: Configure the driver
 
 On Windows, the C++ driver is configured as follows (adjusting the path of the CMake executable as appropriate to your system):
```

Comment:
Suggest removing mention of `libmongoc` from this step. I expect users following this tutorial will not need to install the C driver manually.